scripts/build/macOS/fixup_libs.py

```diff
@@ -6,6 +6,20 @@
 import os
 import shutil
 
+def find_target(path):
```

Comment:
Even though I am not going to approve the first commit at all, I want to stress that naming is very important.

Why is called `find`? It does not perform any kind of searching.

Reply:
Then call it `parse_path` if you like.

Given a path, deduce the target name.